### PR TITLE
[RU] about/index: improvements

### PR DESCRIPTION
**Pull request by lex111**
Hello there!

I am the maintainer of official Russian documentation on PHP, Vue.js, React [JS], ElixirSchool.

I would like _to join_ the i18n team to make the Russian version of the Ruby site better and better, and thus help the Ruby community to popularize this language among Russian speakers.

### CODE REVIEW

File: ru/about/index.md [hunk 1]
```diff
@@ -39,20 +39,20 @@ Ruby-Talk, основная [рассылка](/ru/community/mailing-lists/) д
...
 изменений, копирования и распространения.
 
 ### Всё в Ruby – объекты.
 
 Вначале Matz рассматривал другие языки в поисках идеального синтаксиса.
-Вспоминая свои изыскания, он говорил: "Мне нужен был скриптовый язык,
+Вспоминая свои изыскания, он говорил: «Мне нужен был скриптовый язык,
 который был бы более мощным, чем Perl, и более объектно-ориентированным,
-чем Python."<sup>[2](#fn2)</sup>.”
+чем Python.»<sup>[2](#fn2)</sup>.”
```
Comment (yous) [inline ru/about/index.md]: I think the `."` between "Python" and "\<sup>" was misplaced. See [the source in `en`](https://github.com/ruby/www.ruby-lang.org/blame/fa0136bcab5d6bd3662f8300762077c44fb1ec6e/en/about/index.md#L53).

```suggestion
чем Python<sup>[2](#fn2)</sup>.»
```

**Comment by JuanitoFatas** [comment]
@lex111 Welcome 🎉 Thanks for the contribution. 

@hsbt can invite you to the team.

**Comment by lex111** [comment]
Thank you! I would like to join your friendly team, especially since I recently saw that the Russian translation is no longer maintained by anyone :cry: 

I have a question about how updates occur, i.e. how can I find out what changes ( like diff in git) have occurred in the original document, is this not automated in any way? I mean, do I need to manually look up the changes between the original file and its translation?

**Comment by JuanitoFatas** [comment]
>  I recently saw that the Russian translation is no longer maintained by anyone 😢

I share the same sentiment with you 😭 

The Russian team could be found [at wiki](https://github.com/ruby/www.ruby-lang.org/wiki/Team#i18n-maintainers).

We currently do not have an automated process to find changes easily 😓 . Korean team created this master issue to keep track of upstream changes: https://github.com/ruby/www.ruby-lang.org/issues/1894 that works. 

We could think of ways to automate this, open to suggestions 🙇

**Comment by lex111** [comment]
I mean, in GitHub, you can create a team in an organization, like for example in the React community (see [screenshot](http://joxi.net/krDLw95SKLvwnr)).

**Comment by JuanitoFatas** [comment]
@hsbt could help create a team for you.

**Comment by lex111** [comment]
As an option for tracking changes, you can add the last hash of the commit (you can think of it as a version) of each specific file (original and translation), and then compare these hashes, and thus simplify the identification of what changes had made.

en/about.md

```
---
layout: page
title: "About Ruby"
lang: ru
commit: 1234
---

...
```

ru/about.md

```
---
layout: page
title: "About Ruby"
lang: ru
commit: 1233
---

...
```

Here it is! The hash of the commit of the translated file is different from the hash of the commit of the original file, so the translation needs to be updated!

Did you get my idea?

**Comment by lex111** [comment]
As a continuation of this, so that people know the translation of which files need to be updated, you can create a similar table with translation progress, as done in Elixir School - https://elixirschool.com/ja/report/ (in the case of the Japanese version).

**Comment by lex111** [comment]
Any news?

**Comment by JuanitoFatas** [comment]
@lex111 Thanks for the suggestion. I created #2033 to implement the progress you mention. And thanks for the contributions!

**Comment by lex111** [comment]
I would like to do Russian translation on an ongoing basis, can I become the maintainer, especially now there is no one? Become part of your team.

**Comment by lex111** [comment]
I first of all want to improve the existing translation, and then update it in accordance with the new changes. And then translate the new pages of the site.

**Comment by arbox** [comment]
@lex111 
There were a couple of people doing Russian translations, e.g. @razum2um and @gazay. Did you try to talk to them first?

**Comment by lex111** [comment]
@arbox no, because Ruby team have marked the Russian translation as unmaintained, it means that they can no longer do this anymore, right? :(

**Comment by gazay** [comment]
Hey! It would be great if it's possible to add you as maintainer of Russian version as I don't have time to maintain it anymore. But I don't have rights to do that, we need to ask and wait @hsbt for that.

**Comment by arbox** [comment]
@lex111 I'll help you to maintain the Russian version since the team must consist of at least two people.

@hsbt, what do you think?

**Comment by razum2um** [comment]
Hi @lex111 I'm still alive 😄
Just reviewed your latest submissions, thanks a lot!

**Pull request merged**